contributors/guide/owners.md

```diff
@@ -115,6 +115,35 @@ emeritus_approvers:
 - emily    # 2019-01-05
 ```
 
+#### Cleanup
+
+In addition to Emeritus process above, from time to time it is necessary to
+prune folks from the OWNERS files who are not active in the community. This
```

Comment:
```suggestion
prune folks from the OWNERS files who are not active in the community or specific repository. This
```

We may want to split some repos, especially for reviewers. For example, sig node approvers may be approvers in test-infra sig node areas, but not necessarily a reviewers there.